__tests__/gui/common_cy_functions/general.js

```diff
@@ -314,3 +314,59 @@ export function compareImages(image1, image2) {
 
     expect(pixelDiff).to.equal(0);
 }
+
+export function confirmNoChoroplethFilterSelected() {
+    cy.get(`${mapBottomItems} .map-options .map-options__filter-row:visible`).should('have.length', 1);
+    cy.get(`${mapBottomItems} .map-options .map-options__filter-row:visible .mapping-options__filter`)
+        .eq(0)
+        .find(' .dropdown-menu__selected-item .truncate')
+        .should('have.text', 'Select an attribute');
+    cy.get(`${mapBottomItems} .map-options .map-options__filter-row:visible .mapping-options__filter`).eq(1).should('have.class', 'disabled');
+}
+
+export function confirmNoChartFilterSelected() {
+    cy.get('.rich-data-content .profile-indicator__filter-row:visible').should('have.length', 1);
+    cy.get('.rich-data-content .profile-indicator__filter-row:visible .profile-indicator__filter')
+        .eq(0)
+        .find(' .dropdown-menu__selected-item .truncate')
+        .should('have.text', 'Select an attribute');
+    cy.get('.rich-data-content .profile-indicator__filter-row:visible .profile-indicator__filter').eq(1).should('have.class', 'disabled');
+}
+
+export function selectChoroplethDropdownOption(option, dropdownIndex) {
+    cy.get(`${mapBottomItems} .map-options .map-options__filter-row:visible .mapping-options__filter`)
+        .eq(dropdownIndex)
+        .should('not.have.class', 'disabled');
+    cy.get(`${mapBottomItems} .map-options .map-options__filter-row:visible .mapping-options__filter`)
+        .eq(dropdownIndex)
+        .click();
+
+    cy.get(`.dropdown-menu__content:visible .dropdown__list_item:visible:contains("${option}")`, {timeout: 20000}).click({force: true})
```

Comment:
Cypress docs say the default timeout is 4 seconds, why do we need to change this to 20?

https://docs.cypress.io/guides/references/configuration#Timeouts

Reply:
because the snackbar sometimes block the dropdown options. by increasing the timeout we make sure, in the worst case scenario, cypress will wait until the snackbar is gone